network/network_linux.go

```diff
@@ -476,6 +476,14 @@ func (nm *networkManager) connectExternalInterface(extIf *externalInterface, nwI
 		return err
 	}
 
+	//fetch ovs port number and add it to nwInfo struct
```

Comment:
this code shouldn't be here.. it should be in ovs specific file

Reply:
will move this to AddL2Rules() in ovs_networkclient_linux.go